.github/actions/pr-to-update-go/pr_to_update_go/go_pr_maker.py

```diff
@@ -316,18 +327,18 @@ def set_go_version(self, go_version: str, commit_message: str,
 		print(f'Updated {go_version_file} on {self.repo.name}')
 		return commit
 
-	def update_golang_org_x(self, previous_commit: Commit) -> Union[GitCommit, None]:
+	def update_golang_org_x(self, previous_commit: Commit) -> Optional[GitCommit]:
 		"""
 		:param previous_commit:
 		:type previous_commit:
 		:return:
-		:rtype: Union[GitCommit, None]
+		:rtype: Optional[GitCommit]
```

Comment:
fwiw, I wouldn't bother adding return and parameter types like this when that information is already available in the annotations.

Reply:
Removed typing docstrings in 4eccc0bd80.